### DISCUSSION TITLE
Dark Mode for Artisan

### DISCUSSION THREAD

**Discussion by KodakC41** [artisan-roaster-scope/artisan#1038] [Ideas]
# Dark Mode

I believe that Artisan would benefit from a dark mode for users who are roasting, cupping, or profiling in the evening. I would be happy to implement this. 

 -*C*

**Comment by MAKOMO** [comment]
Dark mode support on Windows seems not to come to Qt bevor v6.5. 

https://bugreports.qt.io/browse/QTBUG-72028

Do you have any more information?
